ironfish-cli/src/commands/deposit.ts

```diff
@@ -28,9 +28,8 @@ export default class Bank extends IronfishCommand {
 
   static flags = {
     ...RemoteFlags,
-    fee: Flags.integer({
+    fee: Flags.string({
```

Comment:
why this change? It will remove some type-checking by the CLI command parser.

Reply:
wanted to sync the same types across different files. accounts/pay.ts uses string for the fee flag. I'll revert this, leave this refactoring for another task.